### PR TITLE
feat: update Radio component to Diete v5

### PR DESCRIPTION
**Pull request by hlavkat**

### CODE REVIEW

**Review by vojtechsimetka — APPROVED** [review]
LGTM, @agazso please give final review

**Review by agazso — COMMENTED** [review]
It HTML the radio is not a standalone component but is used in a group to express mutually exclusive choice. Therefore the focus works in a way that the whole group receives the focus and then you can move between the items.

I know that this PR implements what was designed, but I would argue that maybe the design needs reconsideration as well as the implementation. I would suggest to model the radio component as a group of mutually exclusive selections instead of individual components. See also #84

**Comment by david-gauquelin** [comment]
You're right, I will update the design. Standalon radio makes no sense :)

**Comment by vojtechsimetka** [comment]
OK so we wait for the new design.

**Comment by david-gauquelin** [comment]
The design is done, ready in diete05

File: src/lib/components/ui/radio-button/radio-list.svelte [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
```
Comment (vojtechsimetka) [inline src/lib/components/ui/radio-button/radio-list.svelte:1]: @agazso Is `RadioControl` better name than `RadioList`?

**Review by vojtechsimetka — APPROVED** [review]
Please do the final review @agazso

**Comment by vojtechsimetka** [comment]
One side effect in the current store design is, that changing focus also changes the value. If this ever is a problem, we can change it similarly to how this is done in select.

**Review by vojtechsimetka — CHANGES_REQUESTED** [review]
Two more things
- [ ] format helper text
- [ ] correctly size the label (I believe small label should be small paragraph but unsure)

File: src/lib/components/ui/radio-button/radio-list.svelte (hunk 1 above)
Comment (agazso) [reply src/lib/components/ui/radio-button/radio-list.svelte:1]: This is usually called `RadioGroup`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

**Review by vojtechsimetka — APPROVED** [review]

File: src/lib/components/ui/radio-button/radio-list.svelte (hunk 1 above)
Comment (vojtechsimetka) [reply src/lib/components/ui/radio-button/radio-list.svelte:1]: Sounds good, I'll rename

**Review by agazso — APPROVED** [review]

**Pull request merged**